client/app/locales/en_US.json

```diff
@@ -284,23 +284,23 @@
   "document.pagination.add": "Add new page",
   "document.pagination.delete": "Delete current page",
 
-  "annotations.label.none": "No annotations yet. Add your own!",
-  "annotations.label.help": "Highlight some text to add your own annotations.",
-  "annotations.label.count": "{{count}}",
-  "annotations.label.collaborators": "{count} {count, plural, one{collaborator} other{collaborators}}",
-  "annotations.label.comments": "{count} {count, plural, one{comment} other{comments}}",
-  "annotations.label.replies": "{count} {count, plural, one{reply} other{replies}}",
-  "annotations.label.replies.none": "No replies",
-  "annotations.label.comments.none": "No comments yet.",
-  "annotation.label.jumpto": "Jump to {{title}}",
+  "notes.label.none": "No annotations yet. Add your own!",
+  "notes.label.help": "Highlight some text to add your own annotations.",
+  "notes.label.count": "{{count}}",
+  "notes.label.collaborators": "{count} {count, plural, one{collaborator} other{collaborators}}",
+  "notes.label.comments": "{count} {count, plural, one{comment} other{comments}}",
+  "notes.label.replies": "{count} {count, plural, one{reply} other{replies}}",
+  "notes.label.replies.none": "No replies",
+  "notes.label.comments.none": "No comments yet.",
+  "note.label.jumpto": "Jump to {{title}}",
```

Comment:
This was almost certainly a typo from me, and should be "notes" instead of "note".